### PR TITLE
git-cherry-pick: add example for -n option

### PR DESCRIPTION
**Pull request by waldyrious**
Also format command in main description with backticks.

- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/git-cherry-pick.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # git cherry-pick
 
 > Apply the changes introduced by existing commits to the current branch.
-> To apply changes to another branch, first use git-checkout to switch to the desired branch.
+> To apply changes to another branch, first use `git checkout` to switch to the desired branch.
```
Comment (waldyrious) [inline pages/common/git-cherry-pick.md]: I'm not sure this line should be here at all, actually. It sounds obvious to me, and not particularly related to the `cherry-pick` subcommand, but maybe I'm getting too used to git's tough love 😄

Any opinions on whether to keep it or remove it?

Comment (waldyrious) [reply pages/common/git-cherry-pick.md]: History laughs in my face. Here's a quote from a comment I wrote [back in 2016](https://github.com/tldr-pages/tldr/pull/1196#issuecomment-266944024), in the PR that introduced this page:

> Would it make sense to add another example, say, to cherry pick into a branch that isn't the current one?

To be fair, though, I think that might have been asking for the ability to transport a commit to some place without first going there, but if that's not possible, then I'm not sure the extra line helps much. Thoughts?

Comment (sbrl) [reply pages/common/git-cherry-pick.md]: Hrm, I'm not sure. The change made here is a good one - but I think we should keep this line, as tldr-pages is aimed at beginners - and this is likely to be a common question with respect to the `cherry-pick` subcommand.

Comment (waldyrious) [reply pages/common/git-cherry-pick.md]: Sure, then. Thanks for the input :)

**Pull request merged**